content/pension_pot_options.md

```diff
@@ -202,16 +295,15 @@ You can mix your options at different times in your retirement. For example, you
 
 Pension providers are developing new retirement income products all the time, some of which might mix these options.
 
-$E
```

Comment:
You can avoid having to fall back to HTML if you use Kramdown's inline attribute lists:

```markdown
{: .example--mix }
$E
**Example**

Your pension pot is £60,000. You decide to take 25% as tax-free cash which is £15,000.

{: .example__annuity}
You use £20,000 to buy an annuity. This gives you a taxable income for the rest of your life of about £900 a year.

{: .example__flexible}
You also put £25,000 into a drawdown fund and take out about £5,000 a year for about 5 years.
$E
```

The `{: .foo}` specifies attributes to add to the preceding element. So in this case we're adding a class of `example--mix` to the example block. Later we're adding classes of `example__annuity` and `example__flexible` to the paragraphs.